plugins/lookup/conjur_variable.py

```diff
@@ -203,12 +210,24 @@ def _fetch_conjur_variable(conjur_variable, token, conjur_url, account, validate
     return {}
 
 
+def _store_in_file(value):
+    file_name = ''.join(choice(string.ascii_uppercase + string.digits) for i in range(12))
+    file_path = os.path.join("/dev/shm", file_name)
```

Comment:
`/dev/shm` is not guaranteed to exist on the target system

Reply:
I created a function to return the best possible path, inspired by [Summon equivalent](https://github.com/cyberark/summon/blob/9ca12dcef1bbf7828bdfac5a9e0eac2a44cd3bc3/internal/command/temp_factory.go#L29)